### DISCUSSION TITLE
Odigos autoscaler not working

### DISCUSSION THREAD

**Discussion by satishrajuuvr** [odigos-io/odigos#1660] [Q&A]
when i try to collect traces in kubernetes environment. i  am facing  odigos autoscaler failing with error :  odigos autoscaler unable to migrate collectors workload to new labels

**Comment by RonFed** [comment]
Hi, can you please provide the following details:
1. Which version of Odigos are you using?
2. Do you upgrade from a previous version or Odigos or is this a fresh install?
3. Can you please attach the full log containing the error?
4. Can you also provide more details about your k8s setup and version?

You can also reach out using Slack https://join.slack.com/t/odigos/shared_invite/zt-2tpvvlij5-SFcNlMRWPoqLXKKKxa8Hzw
